### DISCUSSION TITLE
Change navbar column width

### DISCUSSION THREAD

**Discussion by jhlegarreta** [pradyunsg/furo#732] [Questions] [answered]
Hi,
thanks for having developed and maintaining this awesome theme.
I have a project that I contribute to where the name of the package is too wide for the navbar column width:
https://whitematteranalysis--213.org.readthedocs.build/en/213/

Is there a way to either reduce the navbar column width or to make the package name font smaller?

I would like to avoid creating a new entire CSS.

Thanks

**Comment by pradyunsg** [comment, marked as answer]
The link is a 404 now but changing content widths isn't something that this theme ever supported and I don't wish to have it do so either.

Making package name smaller is feasible with CSS. Given that you don't want that, I don't think there's any workable solution I can provide.
